templates/client.conf.erb

```diff
@@ -1,7 +1,17 @@
 client
 dev <%= @dev %>
 proto <%= @proto %>
+<% if @connections -%>
+  <%- @connections.each do | connection | -%>
+    <%- %><connection>
+    <%- connection.each do | key, val | -%>
+       <%= key %> <%= val %>
```

Comment:
Should this be left aligned?

Reply:
no there is not need to left align. the config file would look like:
```
<connection>
       remote somehost.somewhere.com
       port 1194
       proto udp
</connection>
<connection>
       remote otherhost
       .....
```
so it's much more readable if not left aligned.

Reply:
Ah I see.